float_data_for_tock.py

```diff
@@ -0,0 +1,153 @@
+print('\nFetched task items from Float.')
+
+"""
+Fetch people data for all users from Float.
+"""
+endpoint = 'people'
+r = requests.get(url + endpoint, headers=headers)
+people_data = json.loads(r.content.lower().strip())
+print('\nFetched people items from Float.')
+
+"""
+Clean people data for all users from Float to remove all but req'd fields.
+"""
+clean_data = list()
+q = len(people_data['people']) - 1
+while q >= 0:
+    clean_data.append(
+        {'im': people_data['people'][q]['im'],
+            'people_id': people_data['people'][q]['people_id']
+        }
+        )
+    q -= 1
+print('\nCleaned Float people data.')
+
+"""
+Pull out relevant elements from task result.
+"""
+task_list = list()
+for people in task_data['people']:
+    for tasks in people['tasks']:
+        task_list.append(dict(tasks))
+
+for i in task_list:
+    i.update({'tock_pk': None, 'im': None})
+print('\nCleaned Float task data.')
+
+"""
+Fetch Tock Data
+"""
+r = requests.get('https://tock-app.18f.gov/api/users.json')
+output = json.loads(r.content)
+print('\nTock data fetched.')
+
+"""
+Build crosswalk.
+"""
+v = len(output['results']) - 1
+crosswalk = list()
+while v >= 0:
+    crosswalk.append(
+        {'username': output['results'][v]['username'],
+        'pk': output['results'][v]['id']
+        }
+        )
+    v -= 1
+
+a = len(crosswalk) - 1
+b = len(clean_data) - 1
+while a >= 0:
+    while b >= 0:
+        tock_record = str(crosswalk[a]['username'])
+        float_record = str(clean_data[b]['im'])
+        if tock_record[0:12] == float_record[0:12]:
+            crosswalk[a].update({
+                'people_id': clean_data[b]['people_id'],
+                'im': clean_data[b]['im'],
+                }
+                )
+        b -= 1
+    a -= 1
+    b = len(clean_data) - 1
+print('\nFloat / Tock crosswalk built.')
+
+clean_crosswalk = list()
+for i in crosswalk:
+    if len(i)>3:
+        clean_crosswalk.append(i)
+
+x = len(clean_crosswalk)
+print('\nFound %s matches between Float "im" values and Tock "username" '
+    'values.\nAvailable Tock User pk values will be assigned to Float task '
+    'data.') % (x)
+
+"""
+Load values from crosswalk.
+"""
+b = len(clean_crosswalk) - 1
+e = len(task_list) - 1
+count = 0
+while b >= 0:
+    while e >= 0:
+        if clean_crosswalk[b]['people_id'] == task_list[e]['people_id']:
+            task_list[e].update(
+                {'tock_pk': clean_crosswalk[b]['pk'], 'im': clean_crosswalk[b]['im']})
+            count += 1
+        e -=1
+
+    b -= 1
+    e = len(task_list) - 1
+print('\nMatches from crosswalk loaded into Float task data.')
+
+"""
+Write elements to new dictionary that matches Django's loaddata fixture req's.
+"""
+fixture = list()
+for item in task_list:
+    fixture.append(dict())
+y = len(task_list)
+x = y - 1
+while x >= 0:
+    fixture[x]["pk"] = x
+    fixture[x]["model"] = "float.floattasks"
+    fixture[x]["fields"] = task_list[x]
+    x = x - 1
```

Comment:
Quick note that while loops are generally frowned on in Python code. Have you seen [`range`](https://docs.python.org/2/library/functions.html#range)?

As order doesn't matter here, the equivalent loop would be
```python
for x in range(y):
    fixture[x]["pk"] = x
    fixture[x]["model"] = "float.floattasks"
    fixture[x]["fields"] = task_list[x]
```


We can do better, though: you can simplify lines 133 through 142 to something like
```python
fixture = []
for idx, task in enumerate(task_list):
    fixture.append({'pk': idx,
                    'model': 'float.floattasks',
                    'fields': task})
```